util/chunk/column.go

```diff
@@ -334,6 +334,14 @@ func (c *Column) SetNulls(begin, end int, isNull bool) {
 	}
 }
 
+func (c *Column) SetBytes(rowID int, bs []byte) {
+	i := c.offsets[rowID]
+	for _, b := range bs {
+		c.data[i] = b
+		i++
+	}
```

Comment:
We should use `copy(c.data[c.offsets[rowID]:c.offsets[rowID+1], bs)` to limit the size of bs

Reply:
done, PTAL @SunRunAway